### DISCUSSION TITLE
Access Flood from remote system

### DISCUSSION THREAD

**Discussion by spolen88** [jesec/flood#114] [Q&A] [answered]
Type: Question

# Info
I installed Flood with npm, and was unable to access it from my home PC using `my-domain:3000`. I used VNC to connect to my server, and was able to set it up from there using `127.0.0.1:3000` , after setting up Flood on the server, I attempted to access again from home but the page still won't load. I use UFW for my firewall and I have allowed access from my home IP only, using the command `sudo ufw allow from "MY IP ADDRESS" proto tcp to any port 3000`, as I have with all my other web apps. 

# Question
As for my question: What am I doing wrong, is there any way to access this remotely without VNC or ssh tunnels?

**Comment by jesec** [comment, marked as answer]
By default, Flood listens to `127.0.0.1` only. 

To access Flood remotely, you have to listen on appropriate address. Check `-h` argument. You can use `-h 0.0.0.0` to listen on all interfaces.

**Comment by spolen88** [comment]
Thank you for the amazingly quick response, this was exactly what I needed.
